kitsune/questions/utils.py

```diff
@@ -45,12 +44,4 @@ def get_mobile_product_from_ua(user_agent):
         return "ios"
 
     # android
-    try:
-        # We are using firefox instead of Firefox as lower() has been applied to the UA
-        mobile_client = re.search(r"firefox/(?P<version>\d+)\.\d+", ua).groupdict()
-    except AttributeError:
-        return None
-    else:
-        if int(mobile_client["version"]) >= 69:
-            return "firefox-preview"
```

Comment:
We need to update https://github.com/mozilla/kitsune/blob/db42cf7/kitsune/questions/tests/test_utils.py#L111-L112 to reflect the new intended behaviour.